Casks/s/status.rb

```diff
@@ -0,0 +1,34 @@
+cask "status" do
+  arch arm: "aarch64", intel: "x86_64"
+
+  version "2.33.2,16fe57"
+  sha256 arm:   "ee88d0517635dbb5d9469d9c84333215d7b3ece0f4404100794f55fd7c7bf649",
+         intel: "719c8c499ad4a552eab25b10808250b89774a8e66dbf866278cf94329b5b22d0"
+
+  url "https://github.com/status-im/status-desktop/releases/download/#{version.csv.first}/StatusIm-Desktop-#{version.csv.first}-#{version.csv.second}-#{arch}.dmg",
+      verified: "github.com/status-im/status-desktop/releases/download/"
+  name "Status"
+  desc "Decentralised wallet and messenger"
+  homepage "https://status.app/"
+
+  livecheck do
+    url :url
+    regex(/^StatusIm[._-]Desktop[._-](\d+(?:\.\d+)+)[._-](\h+)[._-]#{arch}\.dmg$/i)
```

Comment:
```suggestion
    regex(/^StatusIm[._-]Desktop[._-]v?(\d+(?:\.\d+)+)[._-](\h+)[._-]#{arch}\.dmg$/i)
```

We should still keep the `v?` to conform to our standard pattern.

Reply:
There's no harm in adding extra flexibility to the `regex`. Version bumps will fail either way, but issues may go unnoticed for longer if the `livecheck` incorrectly returns an older version.

Reply:
We would rather see failure at the download step due to the file name changing than new versions not being detected automatically.

Reply:
Added back!